keps/sig-instrumentation/1668-trace-popagating/kep.yaml

```diff
@@ -0,0 +1,47 @@
+title: trace information popagation
+kep-number: 1668
+authors:
+ - "@hase1128"
+ - "@KobayashiD27"
+ - "@fenggw-fnst"
+ - "@zhijianli88"
+ - "@Hellcatlk"
+owning-sig: sig-instrumentation
+participating-sigs:
+  - sig-api-machinery
+status: provisional
```

Comment:
If you want to target this at 1.21, this will need to be set to "implementable" (which I think should trigger the PRR review)

Reply:
Does it mean we have to get approval first? (I checked [kep-workflow](https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/0000-kep-process#kep-workflow))